packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -6,9 +6,8 @@
         data-test-id="countrySelector-toggle"
         v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
         @keyup.esc="closeCountrySelector">
-        <f-button
+        <button
```

Comment:
Changed back to regular button because f-button icon is now not suitable for flag button as supposed to use just simple icons like arrows, crosses etc

Reply:
I think one of the pie-designs for f-button includes having a button with an icon alongside it, but we need to amend the styles of f-button to do this.

[Designs are here](https://app.abstract.com/projects/da082ad2-0d14-40e8-a365-8eca471246e9/branches/master/commits/a60ab83f64456eca407dca472587b118d1a176f7/files/7daa54bc-5a0f-4634-ae91-82ebae259f3c/layers/D1D03086-5F69-421E-8591-33B6FD0F9D90?collectionId=7cce2661-005a-464f-9829-d70715caaffb&collectionLayerId=8ceb3ee2-1feb-4c91-b0f6-51783d824245) (you can see the `button with icon` design on these).

Would implementing these in f-button help potentially?

Reply:
@ashleynolan the problem is not with button with icon and text but with just an icon. When I tried to use f-button for it flag icon became blue. The behavior is correct for icons like cross, arrow, plus etc but not for flag icon.

regular button:
![Screenshot 2021-02-24 at 12 16 48](https://user-images.githubusercontent.com/19548183/108999463-3e779e80-769a-11eb-9394-5d5ef2bbdcf5.png)

f-button with isIcon prop:
![Screenshot 2021-02-24 at 12 30 57](https://user-images.githubusercontent.com/19548183/109000958-3caeda80-769c-11eb-9ea4-f0f0e30c7665.png)

Reply:
Ah ok – I think this is an error in `f-button` – for the ghost type it shouldn't apply a text colour ([which it does here](https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-button/src/components/Button.vue#L357)). So if we fix this in future you should be able to use that component again then.